### PR TITLE
Some Readme content edits

### PR DESCRIPTION
**Pull request by mjbear**
Overall some minor items in the Readme content.

* Fix typo(s)
* Change quoted area to an Alert (a.k.a. admonition)
* Replace a broken netdevops.me hyperlink with an archive.org copy of that page
* Move the Connection Mode list into a table
* Various wording and sentence changes (admittedly my take and can be adjusted)
* Markdown linting-esque changes
  * (full MD linting and GH Actions for another PR!)
  * Surround MD headers with white space
  * Break long MD lines into shorter lines

I'm open to feedback and any tweaks/changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,25 +19,38 @@ Yes, the term is bloated, what it actually means is that with the changes we mad
...
 
-> Although the changes we made here are of a general purpose and you can run vrnetlab routers with docker CLI or any other container runtime, the purpose of this work was to couple vrnetlab with containerlab.  
-> With this being said, we recommend the readers to start their journey from this [documentation entry](https://containerlab.srlinux.dev/manual/vrnetlab/) which will show you how easy it is to run routers in a containerized setting.
+> [!IMPORTANT]
+> Although the changes we made here are of a general purpose and you can run
+> vrnetlab routers with docker CLI or any other container runtime, the purpose
+> of this work was to couple vrnetlab with containerlab.
+>
+> With this being said, we recommend the readers to start their journey from
+> this [documentation entry](https://containerlab.srlinux.dev/manual/vrnetlab/)
```
Comment (hellt) [inline README.md:28]:
```suggestion
> this [documentation entry](https://containerlab.dev/manual/vrnetlab/)
```

File: README.md [hunk 2]
```diff
@@ -19,25 +19,38 @@ Yes, the term is bloated, what it actually means is that with the changes we mad
...
+The default option that we use in containerlab for this setting is `connection-mode=tc`. With this particular mode we use **tc-mirred** redirects to stitch container's interfaces `eth1+` with the ports of the qemu VM running inside.
 
 ![tc](https://gitlab.com/rdodin/pics/-/wikis/uploads/4d31c06e6258e70edc887b17e0e758e0/image.png)
 
-Using tc redirection we get a transparent pipe between container's interfaces and VM's.
+Using tc redirection (tc-mirred) we get a transparent pipe between container's interfaces and VM's.
 
-We scrambled through many alternatives, which I described in [this post](https://netdevops.me/2021/transparently-redirecting-packets/frames-between-interfaces/), but tc-redirect works best of them all.
+We scrambled through many alternatives, which I described in
+[this post](https://web.archive.org/web/20220621131105/https://netdevops.me/2021/transparently-redirecting-packets/frames-between-interfaces/)
```
Comment (hellt) [inline README.md:41]:
```suggestion
[this post](https://netdevops.me/2021/transparently-redirecting-packetsframes-between-interfaces/)
```

**Review by hellt — CHANGES_REQUESTED** [review]
thanks @mjbear 
a couple of small additions

**Comment by mjbear** [comment]
> thanks @mjbear a couple of small additions

Thank you @hellt
I'm going to give the documentation another read or two. I'll mark it as "Ready" when I have it ready for merging.
Thank you!

**Comment by mjbear** [comment]
@hellt
Admittedly my take on some of the sentences and wording. Hopefully I didn't distort the intended information and message.

Just like your earlier suggestions, I'm open to them. :grinning:

**Comment by hellt** [comment]
thank you

**Comment by mjbear** [comment]
> thank you

Teamwork!
Thank you as well. :grinning:

**Pull request merged**